concrete/src/Form/Service/Widget/DateTime.php

```diff
@@ -302,12 +302,15 @@ public function date($field, $value = null, $calendarAutoStart = true)
         if ($requestValue !== null) {
             $dateTime = $requestValue;
         } elseif ($value) {
-            if (!$value instanceof PHPDateTime) {
-                try {
-                    $dateTime = $dh->toDateTime($value);
-                } catch (Exception $x) {
-                }
-            }
+            if ($value instanceof PHPDateTime) {
+              $dateTime = clone $value;
```

Comment:
In case of dates without hours, we must not convert the time zone.
For instance, let's assume that
- we're handling the birth date of Gandhi ('1869-10-02')
- the system has set a time zone of `UTC+0` (for instance. the server is in London)
- the user time zone is `UTC-8` (for instance. the user is in Portland)

If you convert the time zone from the server one to the user one, you'll end subtracting 8 hours to the date, thus having a date of '1869-10-01T14.00', and when we show only the date part we'll have '1869-10-01'.
That's the reason why we don't change the timezone in case of date-only values...